packages/alpine.rb

```diff
@@ -0,0 +1,22 @@
+require 'package'
+
+class Alpine < Package                 # The first character of the class name must be upper case
+  description 'An easy-to-use email client.'
+  homepage 'http://alpine.x10host.com/alpine/'
+  version '2.22'
+  source_url 'http://alpine.x10host.com/alpine/release/src/alpine-2.22.tar.xz'
+  source_sha256 '849567c1b6f71fde3aaa1c97cf0577b12a525d9e22c0ea47797c4bf1cd2bbfdb'   # Use the command "sha256sum"
+
+  def self.build                   # the steps required to build the package
+    system "./configure", "--with-readline", "--prefix=#{CREW_PREFIX}", "--libdir=#{CREW_LIB_PREFIX}", "--with-ssl-dir=/etc/ssl", "--with-ssl-include-dir=/usr/local/include", "--with-ssl-lib-dir=/usr/local/lib64", "--disable-nls"
```

Comment:
We have a set of constants that are used to refer to commonly used paths such as `/usr/local/` (for prefix), `/usr/local/lib` or `/usr/local/lib64` (for libraries), etc. Please look at other packages for how we used them and look at the output of the `crew const` command.

An example would be: `--prefix=/usr/local` should be `--prefix=#{CREW_PREFIX}`

You may also want to look at the PKGBUILD from Arch Linux for this to get an idea of any other configure options that might make sense such as an initial configuration file.